### PR TITLE
Added entity data mapper generator and test for generation

### PR DESCRIPTION
**Pull request by bohdan-harniuk**
**Description** (*)
**What was done:**

1. added entity data mapper generator
2. added test for entity data mapper generation
3. stub for getting data mapper class FQN (used for getListQuery class) changed to the real data mapper class FQN

Screenshot 2021-02-02 at 16 41 15
Screenshot 2021-02-02 at 16 41 27

**Contribution checklist** (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with integration/functional tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: resources/fileTemplates/code/Magento Entity Data Mapper.php.ft [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+namespace ${NAMESPACE};
+
+#set($uses = ${USES})
+#foreach ($use in $uses.split(","))
+use $use;
+#end
+
+ /**
+ * ${ENTITY_NAME} magento model collection to entity data transfer object array mapper.
```
Comment (VitaliyBoyko) [inline resources/fileTemplates/code/Magento Entity Data Mapper.php.ft:12]: This is not model collection

File: src/com/magento/idea/magento2plugin/magento/files/EntityDataMapperFile.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    /**
+     * Get singleton object of entity data mapper file.
+     *
+     * @param entityName String
+     *
+     * @return EntityDataMapperFile
+     */
+    public static EntityDataMapperFile getInstance(final @NotNull String entityName) {
+        synchronized (EntityDataMapperFile.class) {
```
Comment (VitaliyBoyko) [inline src/com/magento/idea/magento2plugin/magento/files/EntityDataMapperFile.java:35]: No singletons

**Review by VitaliyBoyko — CHANGES_REQUESTED** [review]
Please review my minor notices

File: resources/fileTemplates/code/Magento Entity Data Mapper.php.ft (hunk 1 above)
Comment (bohdan-harniuk) [reply resources/fileTemplates/code/Magento Entity Data Mapper.php.ft:12]: Changed.

File: src/com/magento/idea/magento2plugin/magento/files/EntityDataMapperFile.java (hunk 2 above)
Comment (bohdan-harniuk) [reply src/com/magento/idea/magento2plugin/magento/files/EntityDataMapperFile.java:35]: Fixed, thank you!

**Review by VitaliyBoyko — APPROVED** [review]

**Pull request merged**